pkg/trait/logging.go

```diff
@@ -68,17 +66,6 @@ func (l loggingTrait) Configure(environment *Environment) (bool, error) {
 }
 
 func (l loggingTrait) Apply(environment *Environment) error {
-	if environment.IntegrationInPhase(v1.IntegrationPhaseInitialization) {
```

Comment:
I think running the trait during the `IntegrationPhaseInitialization` phase is not needed anymore :)

Reply:
... and it's gone! :D 

Thanks!